data/save.txt

```diff
@@ -1 +1 @@
-jen;0.0
```

Comment:
whenever possible, I would recommend to delete the data/save.txt before pushing as well.

Reply:
OK! I will do.